src/Evolution/Conservative/UpdatePrimitives.hpp

```diff
@@ -40,11 +40,11 @@ struct UpdatePrimitives {
                     const ArrayIndex& /*array_index*/,
                     const ActionList /*meta*/,
                     const ParallelComponent* const /*meta*/) noexcept {
-    using system = typename Metavariables::system;
-    db::mutate_apply<
-        typename system::primitive_from_conservative::return_tags,
-        typename system::primitive_from_conservative::argument_tags>(
-        typename system::primitive_from_conservative{}, make_not_null(&box));
+    using f =
```

Comment:
Since we usually use `f` for continuation-style metafunctions, maybe the name `PrimFromCon` would be better